drivers/include/st77xx.h

```diff
@@ -331,6 +359,14 @@ extern "C" {
                                         LCD_MADCTL_MX       /**< Horizontal mode */
 #define ST77XX_ROTATION_HORZ_FLIP       LCD_MADCTL_MV | \
```

Comment:
```suggestion
#define ST77XX_ROTATION_HORZ_FLIP       ST77XX_ROTATION_90
```

tbh I don't quite understand why 'flip' is 90° rotation, but we should only have one source of truth here.